sinonjs/formatio
# Use @sinonjs/samsam

This will allow sinon to only install one copy of samsam.

The PR removes AMD support from the main library, moving it to a file in `dist/`. Therefore the `semver:major` label.

#### How to verify - mandatory
1. Check out this branch
1. `rm -rf node_modules && npm install`
1. `npm ls samsam`
1. Observe that the only version of `samsam` is samsam@1.3.0, which comes from `@sinonjs/formatio@2.0.0`, which is used by `@sinonjs/referee@2.5.1` (see example below)

```
npm ls samsam
@sinonjs/formatio@2.0.0 /Users/mroderick/code/sinonjs/formatio
└─┬ @sinonjs/referee@2.5.1
  └─┬ @sinonjs/formatio@2.0.0
    └── samsam@1.3.0
``` 

If you get something like this, it means that once this PR is merged, we can update `@sinonjs/referee` to use next major `@sinonjs/formatio` version, and then there will only be one copy of `samsam` (that is `@sinonjs/samsam`) installed.

#### Checklist for author
